app/app.rb

```diff
@@ -282,7 +282,7 @@ def render_directory(directory)
 		def process_form_data(data, git)
 			case data
 			when Array
-				data.map { |v| process_form_data(v, git) }
+				data.reject { |v| v.is_a?(String) && v.blank? }.map { |v| process_form_data(v, git) }
```

Comment:
Well... I mean, blank is pretty much just `== ""` so I'm not sure if we need a method for that.  But if we do we could have a `blank?` that takes an argument?  I dunno.